File: LNPG_01_TIPOS_E_VARIAVEIS/Java/MaiorityAge.java
package LNPG_01_TIPOS_E_VARIAVEIS.Java;
import java.util.Scanner;

public class MaiorityAge {
    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {
            System.out.print("Digite sua idade: ");
            int age = scanner.nextInt();
            if (age >= 18 ) {
                System.out.println("Parabéns, você já pode ser preso!");
            } else {
                System.out.println("Ainda vai demorar um tempo pra ir para cadeia!");
            }
        } catch (Exception e) {}
    }
}

File: LNPG_01_TIPOS_E_VARIAVEIS/haskel/MaiorityAge.hs
module LNPG_01_TIPOS_E_VARIAVEIS.MaiorityAge where
canBePrisioner :: Int -> [Char]
canBePrisioner age = if age >= 18 then "Pode ser Preso" else "Não pode ser preso"
main :: IO ()
main = do
    let age = 17
    putStrLn ( canBePrisioner age )